deploy/charts/trust-manager/templates/clusterrole.yaml

```diff
@@ -44,24 +44,24 @@ rules:
   - "events"
   verbs: ["create", "patch"]
 
+{{- if .Values.secretTargets.enabled }}
+{{- if .Values.secretTargets.authorizedSecretsAll }}
 - apiGroups:
   - ""
   resources:
   - "secrets"
-  verbs: ["create", "list", "watch"]
-{{- if .Values.secretTargets.enabled }}
-{{- if .Values.secretTargets.authorizedSecretsAll }}
+  verbs: ["get", "list", "create", "patch", "watch", "delete"]
+{{- else if .Values.secretTargets.authorizedSecrets }}
 - apiGroups:
   - ""
   resources:
   - "secrets"
-  verbs: ["get", "update", "delete", "patch"]
-{{- else if .Values.secretTargets.authorizedSecrets }}
+  verbs: ["list", "watch"]
```

Comment:
I know this is probably least-privilege permissions, but it also makes the template more complex. I don't see a major benefit of granting `list`, but not `get`, cluster-wide. The controller needs RBAC to read all secrets cluster-wide if secret-targets is enabled - even if it doesn't use the `get` verb.

Reply:
I moved "get" to the other list.